src/core/lib/transport/transport.h

```diff
@@ -111,10 +111,11 @@ void grpc_transport_move_stats(grpc_transport_stream_stats* from,
 // currently handling the batch).  Once a filter or transport passes control
 // of the batch to the next handler, it cannot depend on the contents of
 // this struct anymore, because the next handler may reuse it.
-typedef struct {
-  void* extra_arg;
+struct grpc_handler_private_op_data {
+  void* extra_arg = nullptr;
   grpc_closure closure;
-} grpc_handler_private_op_data;
+  grpc_handler_private_op_data() { memset(&closure, 0, sizeof(closure)); }
```

Comment:
I avoided `memset(0)` for `closure`s in general because the caller always have to initialize closures. It makes a measurable difference in perf.

Reply:
Right, but in this case, I am simply shifting one bigger memset into this. There's memset in closures all over the codebase, so it's either this, or a much larger change that affects every single instance of closures in the codebase. I'd rather do this now to solve gcc8 build failures, and maybe we can migrate closures later and thus get rid of this memset in the process.

Reply:
Sure, sounds good to me to do  this later. Comment resolved :+1: